embulk-input-google_analytics.gemspec

```diff
@@ -15,8 +15,8 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "httpclient"
   spec.add_dependency "google-api-client", "0.10.1"
-  spec.add_dependency "signet"
-  spec.add_dependency "activesupport" # for Time.zone.parse, Time.zone.now
+  spec.add_dependency "signet", "0.11.0"
+  spec.add_dependency "activesupport", "5.2.3" # for Time.zone.parse, Time.zone.now
```

Comment:
minor, should we leave comment why we lock the version of `signet` and `activesupport ` here?

Reply:
Added